core/src/test/java/hudson/util/LineEndNormalizingWriterTest.java

```diff
@@ -37,6 +37,7 @@
 public class LineEndNormalizingWriterTest {
 
     @Test
+    @SuppressWarnings("deprecation")
```

Comment:
We should just delete this copy of the test, since it was moved to Stapler sources.